reactive/k8s.py

```diff
@@ -159,7 +209,46 @@ def gather_sdn_data():
         return addedcontext
     return {}
 
-def render_files(reldata = None):
+
+@contextmanager
+def chdir(path):
+    '''Change the current working directory to a different directory to run
+    commands and return to the previous directory after the command is done.'''
+    old_dir = os.getcwd()
+    os.chdir(path)
+    yield
+    os.chdir(old_dir)
+
+
+def copy_key(directory, prefix):
+    '''Copy the key from the easy-rsa/easyrsa3/pki/private directory to the
+    specified directory. '''
+    if not os.path.isdir(directory):
+        os.makedirs(directory)
+        os.chmod(directory, 0o770)
+    # The key is not in unitdata it is in the local easy-rsa directory.
+    local_key_path = 'easy-rsa/easyrsa3/pki/private/{0}.key'.format(prefix)
+    key_name = '{0}.key'.format(prefix)
+    # The key should be copied to this directory.
+    destination_key_path = os.path.join(directory, key_name)
+    # Copy the key file from the local directory to the destination.
+    copy2(local_key_path, destination_key_path)
+
+
+def docker_compose_kill_remove(service):
```

Comment:
These really need to be refactored to use the charms.docker utilities exposed via the compose module

```
from charms.docker.compose import Compose
compose = Compose('files/kubernetes')
compose.kill()
compose.rm()
```